user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -377,11 +380,12 @@ DataMiner supports Okta as identity provider as from version 10.1.11. Use Okta's
      ...
    </DataMiner>
    ```
+1. User groups have to exist in DataMiner for users to be added to them (both for Groups claims set to true and false). See [Adding a user group] (Adding_a_user_group) for more info on how to do this
```

Comment:
This line has similar issues as line 245: duplicate step in the procedure (creating groups is already mentioned on line 349), missing empty line, incorrect cross-reference. The change below this also has the same issue as line 250.

Reply:
Moved the line to the correct spot, fixed cross-reference, not really sure what you mean with empty line though